src/content/blog/whats-new-october.mdx

```diff
@@ -0,0 +1,125 @@
+
+- 🍾 [@itsmatteomanf](https://github.com/itsmatteomanf) - $250 - For initiative taken translating our docs into Italian.
+- 🎉 [@alexanderniebuhr](https://github.com/alexanderniebuhr) - $250 - For leadership in development on the Cloudflare adapter.
+- 🥳 [@HiDeoo](https://github.com/hideoo) - $250 - For taking ownership in Starlight maintenance, support and new features.
+- 🎈 [@OliverSpeir](https://github.com/oliverspeir) - $250 - For a significant presence and impact in Discord support.
+- 🎊 [@lilnasy](https://github.com/lilnasy) - $500 - For outstanding contributions to the Astro codebase and community.
+
+### ViteConf
+
+This month, three Astro Core members presented talks at [ViteConf 2023](https://viteconf.org/23/).
+
+Learn more in our [blog post](https://astro.build/blog/viteconf-2023/) and watch [the YouTube playlist](https://www.youtube.com/watch?v=veCxKeLl35A&list=PLqGQbXn_GDmkOsHI7-Wrbv1GgAA4tJZhg) of all the talks.
+
+### Content
+
+[Project: U2 Space Baby](https://www.chrsgrrtt.com/u2-space-baby) - Read about Chris Garrett's companion app for U2’s [residence at the Sphere](https://www.u2.com/splash) and see the [new Space Baby app for U2](https://u2spacebaby.u2.com/) app live in action!
+
+[Code Fix: The SIBA Website's Move from Create-React-App to Astro](https://brittanisavery.com/post/move-siba-to-astro) - Read about converting a legacy CRA project to Astro by Brittani S. Avery
+
+[Translate routes in Astro with Content collection, subpage, optional trailingSlash and base support](https://www.webdesign-sopelnik.de/en/blog/translate-routes-for-astro-content-collections-or-subpages-with-trailingslash-and-base-support/) - A guide to translating routes for content collections and subpages by Artur Sopelnik 
+
+## Showcase
+
+Each week on the Astro Community Call in Discord, we shine the spotlight on projects posted by community members in our `#showcase` channel from the following week. Here's a sample of what we saw this month!
+
+### Tools
+
+- [Darkmatter](https://getdarkmatter.dev/) - A CMS-like interface for your Astro content collections (MacOS)
+- [Frontmatter CMS](https://frontmatter.codes/) -  A Headless CMS right in your code editor - now supports Astro!
+- [Search Astro Documentation Raycast Extension](https://www.raycast.com/ElianCodes/search-astro-docs) - search Astro docs directly within Raycast
+- [Set Up a Static Site With Astro on Kinsta](https://kinsta.com/docs/astro-static-site/) - new guide for setting up a static site with Kinsta’s Application or Static Site hosting.
+
+### Themes
+
+- https://github.com/AminoffZ/catonaut by @aminoff
+- https://github.com/kevinzunigacuellar/astro-supabase by @kevinzunigacuellar
+- https://astro.build/themes/details/engineering-blog/ by @eddyvinck
+
+### Websites
+
+- https://www.noobscience.rocks/ by @noobscience
+- https://fadhelmurphy.github.io/ by @mamangpro
+- https://rohitk06.vercel.app/ by @rohit06
+- https://www.hvlxh.tk/ by @hitesh_dev
```

Comment:
This one is also not a valid GitHub profile, and on Twitter this account has no posts at all, I wonder if it's also wrong? I could find the GitHub profile associated with this project is `@hvlxh`

```suggestion
- https://www.hvlxh.tk/ by @hvlxh
```